analyzer/src/test/kotlin/managers/SpdxDocumentFileTest.kt

```diff
@@ -64,6 +67,14 @@ private val pkgForVcs = SpdxPackage(
     name = "Dummy"
 )
 
+private val createSpdxDocument: SpdxDocument
```

Comment:
It is a bit strange to model this as a property. Just make it a function. As the resulting document is always adapted (by changing packages or external document references), the function could accept this data as parameters and return a properly initialized document.